driver/api/impl/attr.cpp

```diff
@@ -434,15 +434,19 @@ SQLRETURN GetStmtAttr(
 
 extern "C" {
 
+#if defined(__clang__) || defined(__GNUC__) || defined(__GNUG__)
+#   pragma GCC visibility push(default)
```

Comment:
Super surprising that `SQL_API` symbols are not  explicitly exported/visible.

Reply:
Looks like ODBC API conventions [unsurprisingly] came from pre-visibility, or even pre-dynamic libraries era. Specifying [public] visibility in the code causes "inconsistent linkage" when building in MSVC/MDAC environment.